### PR TITLE
Supply Chain overview: Add info about Advisories

### PR DESCRIPTION
**Pull request by armchairlinguist**
* Add info about what "Advisories" are in Supply Chain.
* Update screenshot to Vulnerabilities page
* Update and notifications info

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team

### CODE REVIEW

**Review by s-santillan — APPROVED** [review]
LGTM! Also I think we can merge this asap.

File: docs/semgrep-supply-chain/overview.md [hunk 1]
```diff
@@ -27,21 +27,23 @@ Object.entries(frontMatter).filter(
...
 * The severity of the vulnerability.
 
-Semgrep Supply Chain generates a **finding **when it detects a match. If the dependency's version is within the range and finds the matching code within your codebase, the finding is **reachable**.
+Semgrep Supply Chain generates a **finding** when it detects a match. If the dependency's version is within the range and finds the matching code within your codebase, the finding is **reachable**.
 
 A finding is **unreachable** if the dependency contains a known vulnerability, but the vulnerable matching code is not used in your codebase.
 
 In Semgrep Cloud Platform, specific findings of a dependency and code match are called **usages**. Usages are grouped by their **vulnerability**. Vulnerabilities in Semgrep Supply Chain typically have a CVE number corresponding to the record in the [CVE Program](https://www.cve.org/About/Overview).
 
+Semgrep Cloud Platform also includes a list of **Advisories** for reference. Advisories include all vulnerabilities covered by Semgrep Supply Chain, regardless of whether the related dependency is used in scanned code.
```
Comment (andy-r2c) [inline docs/semgrep-supply-chain/overview.md:45]: "Semgrep Cloud Platform also includes a list of **Advisories**, a database of all disclosed CVEs for open source dependencies in ecosystems covered by Semgrep Supply Chain. This list is independent of whether the related dependency is used in scanned code (See Vulnerabilities)"

**Review by andy-r2c — CHANGES_REQUESTED** [review]

File: docs/semgrep-supply-chain/overview.md (hunk 1 above)
Comment (andy-r2c) [reply docs/semgrep-supply-chain/overview.md:45]: we have a script that automatically generates parity rules as soon as GHSA discloses something new--the writing part only applies to reachability rules

Comment (s-santillan) [reply docs/semgrep-supply-chain/overview.md:45]: Discussed with Andy and made into a new Linear task: https://linear.app/r2c/issue/MKT-601/clarity-on-advisories-transitivity-how-we-work

Resolving.

**Pull request merged**